tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +205,44 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin'] = function() {
+
+		var errorLogStub = sinon.stub( console, 'error' );
+
+		var onErrorStub = function() {
+			resume( function() {
+				assert.fail( 'There should be no RunTime errors!' );
+			} );
+		};
+
+		window.addEventListener( 'error', onErrorStub );
+
+		var instanceReadyCallback = function( e ) {
+			resume( function() {
+				assert.isFalse( errorLogStub.called, 'There should be no errors during initialization' );
+			} );
+		};
+
+		bender.editorBot.create( {
+			name: 'bbcodeFailTest',
+			config: {
+				extraPlugins: 'bbcode',
+				editorplaceholder: 'any',
+				fullPage: true,
+				on: {
+					instanceReady: instanceReadyCallback
+				}
+			}
+		}, function( bot ) {
+
+		} );
```

Comment:
It will be good to check if editor placeholder is present or not (you can see how it's done in other tests). However, due to issue with `bbcode` always returning empty space (see review description) it might be not possible for `bbcode` + `fullpage` combination. This means it will be good to add another test :wink: Probably the one mentioned in https://github.com/ckeditor/ckeditor4/pull/4370#discussion_r520687252 - you can just use fixed content like:

```
editor.on( 'toDataFormat', function( evt) {
	evt.data.dataValue = '<p>Foo Bar</p>'
}, null, null, 16 );
```

Also it's good to have two versions of such tests - one with empty editor, another one with editor preinitliazed with some content. This would mean we should add 4 tests:

*   `bbcode` + `fullpage` with no content (this one we already have)
*   `bbcode` + `fullpage` with preinitialized content
*   `toDataFormat` body filtering with no content
*   `toDataFormat` body filtering with preinitialized content

Reply:
Added test cases with [3d52ae18](https://github.com/ckeditor/ckeditor4/pull/4370/commits/d52ae18718fd2f0f89053aa85727ecdca42be7b9)